+++
authors = ["Derrick E Fouts", "Michael A Matthias", "Haritha Adhikarla", "Ben Adler", "Luciane Amorim-Santos", "Douglas E Berg", "Dieter Bulach", "Alejandro Buschiazzo", "Yung-Fu Chang", "Renee L Galloway", "David A Haake", "Daniel H Haft", "Rudy Hartskeerl", "Albert I Ko", "Paul N Levett", "James Matsunaga", "Ariel E Mechaly", "Jonathan M Monk", "Ana L T Nascimento", "Karen E Nelson", "Bernhard Palsson", "Sharon J Peacock", "Mathieu Picardeau", "Jessica N Ricaldi", "Janjira Thaipandungpanit", "Elsio A Wunder", "X Frank Yang", "Jun-Jie Zhang", "Joseph M Vinetz"]
title = "What Makes a Bacterial Species Pathogenic?:Comparative Genomic Analysis of the Genus Leptospira."
journal = "PLoS neglected tropical diseases"
what = "article"
doi = "10.1371/journal.pntd.0004403"
pubmed = "26890609"
date = "2016-02-20"
keywords = []
+++

Leptospirosis, caused by spirochetes of the genus Leptospira, is a globally widespread, neglected and emerging zoonotic disease. While whole genome analysis of individual pathogenic, intermediately pathogenic and saprophytic Leptospira species has been reported, comprehensive cross-species genomic comparison of all known species of infectious and non-infectious Leptospira, with the goal of identifying genes related to pathogenesis and mammalian host adaptation, remains a key gap in the field. Infectious Leptospira, comprised of pathogenic and intermediately pathogenic Leptospira, evolutionarily diverged from non-infectious, saprophytic Leptospira, as demonstrated by the following computational biology analyses: 1) the definitive taxonomy and evolutionary relatedness among all known Leptospira species; 2) genomically-predicted metabolic reconstructions that indicate novel adaptation of infectious Leptospira to mammals, including sialic acid biosynthesis, pathogen-specific porphyrin metabolism and the first-time demonstration of cobalamin (B12) autotrophy as a bacterial virulence factor; 3) CRISPR/Cas systems demonstrated only to be present in pathogenic Leptospira, suggesting a potential mechanism for this clade's refractoriness to gene targeting; 4) finding Leptospira pathogen-specific specialized protein secretion systems; 5) novel virulence-related genes/gene families such as the Virulence Modifying (VM) (PF07598 paralogs) proteins and pathogen-specific adhesins; 6) discovery of novel, pathogen-specific protein modification and secretion mechanisms including unique lipoprotein signal peptide motifs, Sec-independent twin arginine protein secretion motifs, and the absence of certain canonical signal recognition particle proteins from all Leptospira; and 7) and demonstration of infectious Leptospira-specific signal-responsive gene expression, motility and chemotaxis systems. By identifying large scale changes in infectious (pathogenic and intermediately pathogenic) vs. non-infectious Leptospira, this work provides new insights into the evolution of a genus of bacterial pathogens. This work will be a comprehensive roadmap for understanding leptospirosis pathogenesis. More generally, it provides new insights into mechanisms by which bacterial pathogens adapt to mammalian hosts.