+++
authors = ["Garrett W Birkel", "Amit Ghosh", "Vinay S Kumar", "Daniel Weaver", "David Ando", "Tyler W H Backman", "Adam P Arkin", "Jay D Keasling", "Héctor García Martín"]
title = "The JBEI quantitative metabolic modeling library (jQMM): a python library for modeling microbial metabolism."
journal = "BMC bioinformatics"
what = "article"
doi = "10.1186/s12859-017-1615-y"
pubmed = "28381205"
date = "2017-04-05"
keywords = ["-omics data", "13 C Metabolic Flux Analysis", "Flux analysis", "Predictive biology"]
+++

Modeling of microbial metabolism is a topic of growing importance in biotechnology. Mathematical modeling helps provide a mechanistic understanding for the studied process, separating the main drivers from the circumstantial ones, bounding the outcomes of experiments and guiding engineering approaches. Among different modeling schemes, the quantification of intracellular metabolic fluxes (i.e. the rate of each reaction in cellular metabolism) is of particular interest for metabolic engineering because it describes how carbon and energy flow throughout the cell. In addition to flux analysis, new methods for the effective use of the ever more readily available and abundant -omics data (i.e. transcriptomics, proteomics and metabolomics) are urgently needed.